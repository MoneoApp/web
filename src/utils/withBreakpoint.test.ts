import { css } from '@emotion/react';

import { withBreakpoint } from './withBreakpoint';

const style = css`
  color: #ffba05;
`;

it('should insert the correct breakpoint', () => {
  expect(withBreakpoint('desktop', style).styles).toMatchSnapshot();
});

it('should skip the breakpoint on mobile', () => {
  expect(withBreakpoint('phone', style).styles).toMatchSnapshot();
});
