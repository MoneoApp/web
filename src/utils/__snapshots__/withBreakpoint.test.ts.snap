// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should insert the correct breakpoint 1`] = `"@media (min-width: 93.75rem){color:#ffba05;label:style;;;};label:withBreakpoint;"`;

exports[`should skip the breakpoint on mobile 1`] = `"color:#ffba05;label:style;"`;
